lib_eio_posix/fs.ml

```diff
@@ -161,6 +161,21 @@ end = struct
     Eio.Resource.T (d, Handler.v)
 
   let pp f t = Fmt.string f (String.escaped t.label)
+
+  let native_int t path =
```

Comment:
What's the reasoning for calling this `native_int` is it should for `internal` ? I couldn't help but think of `Stdlib.Nativeint` ^^"

Reply:
Good point - I've renamed it.